master/reference/public-cloud/aws.md

```diff
@@ -0,0 +1,152 @@
+---
+title: AWS
+---
+
+Calico is designed to provide high performance massively scalable virtual
```

Comment:
[Flannel's AWS backend doc](https://github.com/coreos/flannel/blob/master/README.md#backends) states that it still limits you to 50:

>Note: Currently, AWS limits the number of entries per route table to 50.